common/src/main/java/io/netty/util/concurrent/GlobalEventExecutor.java

```diff
@@ -65,6 +65,8 @@ public void run() {
 
     private GlobalEventExecutor() {
         scheduledTaskQueue().add(quietPeriodTask);
+        threadFactory = ThreadExecutorMap.apply(new DefaultThreadFactory(
+                DefaultThreadFactory.toPoolName(getClass()), false, Thread.NORM_PRIORITY, null), this);
```

Comment:
How about moving the decoration into `FastThreadLocalThread`? Then we don't need to wrap here and potentially in many other places.

Reply:
I was thinking about this but I think some people may end up using their own `Thread` and not `FastThreadLocalThread`

Reply:
I see. Then how about keeping it as it is here and modifying `FastThreadLocalThread` as well, with some duplication suppression logic if possible?

Reply:
Actually I think I would like to keep it as it is as otherwise we will need to also pass in the `EventExecutor` to the `FastThreadLocalThread` constructor.